### PR TITLE
Fix/axis labels

### LINKED ISSUE #61: fix/x-axis labels
Currently, x-axis labels for the Line Charts on the Dashboard view are too closely bunched together
Two potential fixes include:
- Removing the labels completely with 
```
xaxis: { 
    labels: { 
        show: false 
        formatter: function(value){
              return "Particpant " + value;
        }
     } 
...}
```
Result: 
987c68d8-d954-4cb7-af5f-c18e8e705125

- Have label contain "|" to create distinction and remove overlapping labels
```
xaxis: { 
  labels: { 
            rotate: 0,
            hideOverlappingLabels: true,
            formatter: function(value){
              return value + " |";
            }
  } 
...}
```
Result: 
0d49b7d0-74b0-4674-81f5-1075d53886a9

### PR DESCRIPTION
**Pull request by LiamSingh64**
This PR should:
- Reduce the amount of labels displayed on the x-axis for the line charts on DashboardView

### CODE REVIEW

**Comment by LiamSingh64** [comment]
Couldn't make it work with `categories` so just used `overwriteCategories` instead.
@rwx-yxu How's this?:
image
image

**Comment by LiamSingh64** [comment]
Could add 
```
title: {
   text: 'Participants'
},
```
to make it more clear to the user that the numbers represent particpants?

File: src/utils/chart.js [hunk 1]
```diff
@@ -97,13 +97,19 @@ export function LineChartBuilder(title,data,label){
...
+        curve: 'straight',
+        width: 2
       },
       dataLabels: {
         enabled: false
       },
       xaxis: {
-        categories: ''
+        type: 'category',
+        overwriteCategories: ['1', (data.length)/4, (data.length)/2, (data.length*3)/4, data.length],
```
Comment (rwx-yxu) [inline src/utils/chart.js:108]: Why call it overwriteCategories when there was already an item called categories before?
By hard coding in it at 1/4, 1/2, 3/4 it doesnt make it flexible if for future use, it is easier to comprehend specific intervals like every 50 or 20. Existing screen shot has it at roughly each 80 which isn't all that common when looking at data.

**Review by rwx-yxu — CHANGES_REQUESTED** [review]
Would suggest to just have a new parameter like xaxisSkip that accepts the intervals that should be marked and default it to something like 10. Then, in the components we can just pass through the interval we want to highlight.

File: src/utils/chart.js (hunk 1 above)
Comment (LiamSingh64) [reply src/utils/chart.js:108]: How's this?
image
image

Comment (LiamSingh64) [reply src/utils/chart.js:108]: and the specific interval is passed through as a parameter

Comment (rwx-yxu) [reply src/utils/chart.js:108]: That's fine. Just make sure to keep the one value at the beginning. Remove the comments.

Comment (LiamSingh64) [reply src/utils/chart.js:108]: Just couldn't get anything working with `categories` for some reason, `overwriteCategories` worked and I have no idea why!

**Review by rwx-yxu — APPROVED** [review]

**Pull request merged**